src/dom.js

```diff
@@ -380,6 +392,10 @@ equalizeNode.impl = (node, root = true, inlineTextNodes = []) => {
     collapseTextAcrossInlineNodes();
   }
 
+  if (node && node.nodeType === node.ELEMENT_NODE && node.className) {
+    node.className = pipe(node.className.split(' '), filter(Boolean), uniq, mapSort(identity), join(' '));
```

Comment:
There is `node.classList` which should save you from performing the filter and the manual splitting…

Reply:
node.classList also deals with various kinds of spaces…

Reply:
In my browser and jsdom classList's value is immutable, the variable can be assigned to like classList.
Although I was suggesting reading from classList and writing to className anyways :)